linera-service/src/exporter/proto/indexer.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+package indexer.linera_indexer;
+
+import "google/protobuf/empty.proto";
+
+service Indexer {
+    rpc IndexBatch(stream Element) returns (stream google.protobuf.Empty) {}
```

Comment:
If you won't change this interface to return a stream of block heights (so that it's clear that we're _ack-ing_ **blocks**) then at least add a very clear comment on this interface for the implementors.